pservice/setup.py

```diff
@@ -60,6 +60,8 @@
 module_path = 'pdo/pservice/enclave'
 module_src_path = os.path.join(script_dir, module_path)
 
+debug_flag = os.environ.get('PDO_DEBUG_BUILD', False) in ("1")
+
```

Comment:
umm... what?

Reply:
I have fixed the "False", it should have been "0", and I added a comment to clarify.
It may be convoluted but the retrieval of the environment variables returns a string (we set PDO_DEBUG_BUILD as "0" or "1"), which has to be converted to a bool. I'm open to suggestions.